### PR TITLE
Feature/35 create logo component

### PR DESCRIPTION
**Pull request by JordiGN**
# Description
What did you implemented/Why did you implemented this:
-Implementation of the component that renders de logo
-It was made customizable in order to change its height and width as needed
 Examples:
 - What?: this PR is related to issue #35 
 - Why?: We detect that the loge is used in different screens and also that it is more than just an image
 
 # Testing
-Download changes
-Open project folder on terminal
-NPM install
-Android
npx react-native run-android
-iOS, could not be tested due to inconvenience on iOS/Xcode
 
  
  ## Screenshots 

image
image

### CODE REVIEW

**Review by Ing-Heber — DISMISSED** [review]
Good job Jordi!!

File: tribomobile/components/LogoTribo.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React, {Component} from 'react';
+import {View, Image, Text, StyleSheet} from 'react-native';
+
+import Tribologo from '../assets/tribologo.png';
+
+const LogoTribo = (props) => {
+  const {height,width,line} = props;
+  return (
+    <View
+      style={{height: '100%', justifyContent: 'center', alignItems: 'center'}}>
```
Comment (LeyaimMagma) [inline tribomobile/components/LogoTribo.js]: Add this on StyleSheet

File: tribomobile/components/LogoTribo.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          width: '80%',
+        }}></View>
+      }
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  icon: {
+    tintColor: '#037D94',
```
Comment (LeyaimMagma) [inline tribomobile/components/LogoTribo.js]: Create a colors.js file and import all the color that you need

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

File: tribomobile/components/LogoTribo.js (hunk 1 above)
Comment (JordiGN) [reply tribomobile/components/LogoTribo.js]: Done

File: tribomobile/components/LogoTribo.js (hunk 2 above)
Comment (JordiGN) [reply tribomobile/components/LogoTribo.js]: you are right, we already had that file, I've just importer to my component

**Review by develiasdaniel — DISMISSED** [review]
Nice design jordi, very clean

**Review by develiasdaniel — DISMISSED** [review]

**Review by Ing-Heber — DISMISSED** [review]

**Review by Ocosmee — DISMISSED** [review]

**Review by tovl1 — DISMISSED** [review]
Good!

**Review by develiasdaniel — APPROVED** [review]

**Review by alcamk — APPROVED** [review]
Okay, hope you can work again

**Review by Ocosmee — APPROVED** [review]
Good job

**Review by LeyaimMagma — APPROVED** [review]

**Pull request merged**